hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/action/rollback/TestMergeOnReadRollbackActionExecutor.java

```diff
@@ -89,7 +88,7 @@ public void tearDown() throws Exception {
   }
 
   @ParameterizedTest
-  @ValueSource(booleans = {true})
+  @ValueSource(booleans = {false, true})
```

Comment:
do we know why this was not having "false" before. may be we had some reason to remove it explicitly.

Reply:
`false` case was removed in https://github.com/apache/hudi/pull/6313

not sure exactly why. but it's passing now.